core/templates/dev/head/pages/exploration-editor-page/services/router.service.ts

```diff
@@ -127,6 +127,7 @@ oppia.factory('RouterService', [
             if (pathType === SLUG_GUI) {
               $rootScope.$broadcast('refreshStateEditor');
             }
+            $rootScope.$broadcast('redrawGraph');
```

Comment:
Hi @kevintab95, some notes/questions:

- How much of a slowdown are you seeing with a large exploration when you do this? (Try loading one of the larger demo explorations.)

- Could we have a more targeted 'centerGraph' event that just centers the graph (rather than traversing the whole exploration and redoing the layout from scratch)? That would probably be better, given the context of the TODO 
- Should we remove the TODO and replace it with a comment above this line explaining why the redrawGraph $broadcast is needed?below.

Reply:
Takes a little less than a second to center the graph with demo explorations having 12 cards. I've moved the centering and panning portion of the code to separate functions, calling centerGraph will trigger just these. Also, replaced the TODO with a comment. PTAL thanks!